migration/sql-files/002-fix-273-workitemtype.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE work_item_types
+SET fields = '{"system.state": {"Type": {"Kind": "enum", "Values": ["new", "in progress", "resolved", "closed"], "BaseType": {"Kind": "string"}}, "Required": true}, "system.title": {"Type": {"Kind": "string"}, "Required": true}, "system.creator": {"Type": {"Kind": "user"}, "Required": true}, "system.assignee": {"Type": {"Kind": "user"}, "Required": false}, "system.description": {"Type": {"Kind": "string"}, "Required": false}}';
```

Comment:
So the way to think of the split been migration and PopulateCommon types would be:

If you didn't run the alm server with --polulateCommon, does the migration make sense to do? If it doesn't, then it belongs under PopulateCommons, else it belongs under migration.

What the proposed migration effectively is doing is resetting all my custom made WorkItemTypes to our Common types. 

What we need to do here is to add an 'insert or update' feature to the Populate Common types. The WIT Repository doesn't currently support it, but for now, we only controlled force it in the database anyway, so we can still do that. If 'system.X' WIT exists, update the fields for 'system.X' but serializing the Fields to the database.